ModelManager.cpp

```diff
@@ -203,15 +217,37 @@ Blob::Ptr NnapiModelInfo::GetInOutOperandAsBlob(RunTimeOperandInfo& op, const ui
                 return blob;
             }
         }
-    } else if (op.type == OperandType::TENSOR_INT32) {
+    } else if (op.type == OperandType::TENSOR_INT32 || op.type == OperandType::INT32) {
         ALOGD("check if const tensors of type IN32 supported");
-        // nnAssert(true);
         InferenceEngine::TensorDesc td(InferenceEngine::Precision::I32, toDims(op.dimensions),
                                        InferenceEngine::Layout::ANY);
-        return std::make_shared<InferenceEngine::TBlob<int32_t>>(td, (int32_t*)buf, len);
-    } else {
-        ALOGD("not supporting const tensors of type ", op.type);
-        nnAssert(false);
+        if (buf == nullptr) {
+            ALOGD("TENSOR_INT32 buf is NULL !!!!!!!!!!!!!!!");
+            InferenceEngine::TBlob<int32_t>::Ptr blob =
+                std::make_shared<InferenceEngine::TBlob<int32_t>>(td);
+            blob->allocate();
+            return blob;
+        } else {
+            InferenceEngine::TBlob<int32_t>::Ptr blob =
+                std::make_shared<InferenceEngine::TBlob<int32_t>>(td, (int32_t*)buf, len);
+            return blob;
+        }
+    } else if (op.type == OperandType::TENSOR_BOOL8 || op.type == OperandType::BOOL) {
+        ALOGD("check if const tensors of type BOOL supported");
```

Comment:
Can you use ALOGV where its not-so-important, as the logcat will be flooded with these logs.